releasenotes/notes/add-primitives-support-31af39549b5e66e3.yaml

```diff
@@ -0,0 +1,24 @@
+---
+prelude: >
+    Qiskit Optimization 0.5 supports the new algorithms introduced in Qiskit Terra 0.22
+    which in turn rely on the `Qiskit Primitives <https://qiskit.org/documentation/apidoc/primitives.html>`_.
+    Qiskit Optimization 0.5 still supports the former algorithms based on ``QuantumInstance``,
+    but they will be deprecated and then removed, along with the support here, in future releases.
+
+features:
+  - |
+    The :class:`~.MinimumEigenOptimizer` class takes the primitives-based algorithms
+    (``qiskit.algorithms.minimum_eigensolvers.SamplingMinimumEigensolver`` and
```

Comment:
Can we change all the highlighted qiskit.algorithm entries here over  to \:class\: references. These should resolve through intersphinx right so that they will be links not just highlighted entries.

Reply:
Sure. I will do it.

Reply:
done